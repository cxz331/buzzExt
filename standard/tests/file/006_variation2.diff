006+ 105777
006- 107777
008+ 45777
008- 47777
026+ 105001
026- 107001
028+ 45001
028- 47001
041+ 105141
041- 107141
043+ 45141
043- 47141
051+ 101567
051- 103567
053+ 41567
053- 43567
056+ 101567
056- 103567
058+ 41567
058- 43567
063+ 101567
063- 103567
067+ 41567
067- 43567
072+ 101567
072- 103567
076+ 41567
076- 43567
081+ 101567
081- 103567
085+ 41567
085- 43567
090+ 101567
090- 103567
094+ 41567
094- 43567