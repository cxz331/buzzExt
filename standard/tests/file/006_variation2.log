
---- EXPECTED OUTPUT
*** Testing fileperms() & chmod() : usage variations ***

*** Testing fileperms(), chmod() with miscellaneous permissions ***
-- Iteration 1 --
bool(true)
107777
bool(true)
47777
-- Iteration 2 --
bool(true)
100000
bool(true)
40000
-- Iteration 3 --
bool(true)
101000
bool(true)
41000
-- Iteration 4 --
bool(true)
101111
bool(true)
41111
-- Iteration 5 --
bool(true)
107001
bool(true)
47001
-- Iteration 6 --
bool(true)
100001
bool(true)
40001
-- Iteration 7 --
bool(true)
101411
bool(true)
41411
-- Iteration 8 --
bool(true)
107141
bool(true)
47141
-- Iteration 9 --
bool(true)
100637
bool(true)
40637
-- Iteration 10 --
bool(true)
103567
bool(true)
43567
-- Iteration 11 --
bool(true)
103567
bool(true)
43567
-- Iteration 12 --

Warning: chmod() expects parameter 2 to be long, string given in %s on line %d
NULL
103567

Warning: chmod() expects parameter 2 to be long, string given in %s on line %d
NULL
43567
-- Iteration 13 --

Warning: chmod() expects parameter 2 to be long, string given in %s on line %d
NULL
103567

Warning: chmod() expects parameter 2 to be long, string given in %s on line %d
NULL
43567
-- Iteration 14 --

Warning: chmod() expects parameter 2 to be long, string given in %s on line %d
NULL
103567

Warning: chmod() expects parameter 2 to be long, string given in %s on line %d
NULL
43567
-- Iteration 15 --

Warning: chmod() expects parameter 2 to be long, string given in %s on line %d
NULL
103567

Warning: chmod() expects parameter 2 to be long, string given in %s on line %d
NULL
43567
*** Done ***
---- ACTUAL OUTPUT
*** Testing fileperms() & chmod() : usage variations ***

*** Testing fileperms(), chmod() with miscellaneous permissions ***
-- Iteration 1 --
bool(true)
105777
bool(true)
45777
-- Iteration 2 --
bool(true)
100000
bool(true)
40000
-- Iteration 3 --
bool(true)
101000
bool(true)
41000
-- Iteration 4 --
bool(true)
101111
bool(true)
41111
-- Iteration 5 --
bool(true)
105001
bool(true)
45001
-- Iteration 6 --
bool(true)
100001
bool(true)
40001
-- Iteration 7 --
bool(true)
101411
bool(true)
41411
-- Iteration 8 --
bool(true)
105141
bool(true)
45141
-- Iteration 9 --
bool(true)
100637
bool(true)
40637
-- Iteration 10 --
bool(true)
101567
bool(true)
41567
-- Iteration 11 --
bool(true)
101567
bool(true)
41567
-- Iteration 12 --

Warning: chmod() expects parameter 2 to be long, string given in /data/service/php-5.5.28/ext/standard/tests/file/006_variation2.php on line 51
NULL
101567

Warning: chmod() expects parameter 2 to be long, string given in /data/service/php-5.5.28/ext/standard/tests/file/006_variation2.php on line 56
NULL
41567
-- Iteration 13 --

Warning: chmod() expects parameter 2 to be long, string given in /data/service/php-5.5.28/ext/standard/tests/file/006_variation2.php on line 51
NULL
101567

Warning: chmod() expects parameter 2 to be long, string given in /data/service/php-5.5.28/ext/standard/tests/file/006_variation2.php on line 56
NULL
41567
-- Iteration 14 --

Warning: chmod() expects parameter 2 to be long, string given in /data/service/php-5.5.28/ext/standard/tests/file/006_variation2.php on line 51
NULL
101567

Warning: chmod() expects parameter 2 to be long, string given in /data/service/php-5.5.28/ext/standard/tests/file/006_variation2.php on line 56
NULL
41567
-- Iteration 15 --

Warning: chmod() expects parameter 2 to be long, string given in /data/service/php-5.5.28/ext/standard/tests/file/006_variation2.php on line 51
NULL
101567

Warning: chmod() expects parameter 2 to be long, string given in /data/service/php-5.5.28/ext/standard/tests/file/006_variation2.php on line 56
NULL
41567
*** Done ***
---- FAILED
