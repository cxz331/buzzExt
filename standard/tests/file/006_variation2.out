*** Testing fileperms() & chmod() : usage variations ***

*** Testing fileperms(), chmod() with miscellaneous permissions ***
-- Iteration 1 --
bool(true)
105777
bool(true)
45777
-- Iteration 2 --
bool(true)
100000
bool(true)
40000
-- Iteration 3 --
bool(true)
101000
bool(true)
41000
-- Iteration 4 --
bool(true)
101111
bool(true)
41111
-- Iteration 5 --
bool(true)
105001
bool(true)
45001
-- Iteration 6 --
bool(true)
100001
bool(true)
40001
-- Iteration 7 --
bool(true)
101411
bool(true)
41411
-- Iteration 8 --
bool(true)
105141
bool(true)
45141
-- Iteration 9 --
bool(true)
100637
bool(true)
40637
-- Iteration 10 --
bool(true)
101567
bool(true)
41567
-- Iteration 11 --
bool(true)
101567
bool(true)
41567
-- Iteration 12 --

Warning: chmod() expects parameter 2 to be long, string given in /data/service/php-5.5.28/ext/standard/tests/file/006_variation2.php on line 51
NULL
101567

Warning: chmod() expects parameter 2 to be long, string given in /data/service/php-5.5.28/ext/standard/tests/file/006_variation2.php on line 56
NULL
41567
-- Iteration 13 --

Warning: chmod() expects parameter 2 to be long, string given in /data/service/php-5.5.28/ext/standard/tests/file/006_variation2.php on line 51
NULL
101567

Warning: chmod() expects parameter 2 to be long, string given in /data/service/php-5.5.28/ext/standard/tests/file/006_variation2.php on line 56
NULL
41567
-- Iteration 14 --

Warning: chmod() expects parameter 2 to be long, string given in /data/service/php-5.5.28/ext/standard/tests/file/006_variation2.php on line 51
NULL
101567

Warning: chmod() expects parameter 2 to be long, string given in /data/service/php-5.5.28/ext/standard/tests/file/006_variation2.php on line 56
NULL
41567
-- Iteration 15 --

Warning: chmod() expects parameter 2 to be long, string given in /data/service/php-5.5.28/ext/standard/tests/file/006_variation2.php on line 51
NULL
101567

Warning: chmod() expects parameter 2 to be long, string given in /data/service/php-5.5.28/ext/standard/tests/file/006_variation2.php on line 56
NULL
41567
*** Done ***